### PR TITLE
Update Licensing Details within README.md

### PR DESCRIPTION
**Pull request by maxwolfs**
specifies the licensing distinction of src and content

### CODE REVIEW

**Review by fdobrovolny — APPROVED** [review]
LGTM

**Review by chrisschwa — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -3,8 +3,8 @@
 ## Licenses
 
 The content of this repository is published under multiple licenses.
-All files within `docs` are licensed under [CC-BY-SA 4.0](LICENSE-DOCS).
-All source code is licensed under the [MIT](LICENSE-CODE) license.
+All documentation content files within `docs, dev-docs, standards and community` are licensed under [CC-BY-SA 4.0](LICENSE-DOCS).
+All source code of this documenation page is licensed under the [MIT](LICENSE-CODE) license.
```
Comment (fkr) [inline README.md:7]: Two reviewers, and neither spots the typo ;) @fdobrovolny @chrisschwa 

"documenation"

Comment (rholling) [reply README.md:7]: Realy strange, because, we talked about that exactly yesterday afternoon....

Comment (fkr) [reply README.md:7]: It was fixed directly afterwards, I was just going through my backlog from yesterday and was amused by that ;)

**Pull request merged**